### ISSUE TITLE
In Glimmer: EmberObject.create no longer supports defining computed properties

### ISSUE THREAD

**Issue by gvocale** [adopted-ember-addons/ember-notify#155]
I am trying to do the following:
```
// notifications/index.ts

import { action } from '@ember/object';
import Component from '@glimmer/component';

import Notify from 'ember-notify';

export default class Notifications extends Component {
  alternativeNotify = Notify.property();

  @action
  shootNotification() {
    this.alternativeNotify.success('Hello from the controller');
  }
}
```
and
```
// notifications/index.hbs

<button type="button" {{on "click" this.shootNotification}}>
  Shoot
</button>
<EmberNotify @source={{this.alternativeNotify}} />
```
But I get the following error in console:
```
Uncaught Error: Assertion Failed: EmberObject.create no longer supports defining computed properties. Define computed properties using extend() or reopen() before calling create().
```

**Comment by gvocale** [comment]
Rewriting it this way seem to have fixed the issue:
```import Notify from 'ember-notify';
import Component from '@glimmer/component';
import { action } from '@ember/object';

export default class MyComponent extends Component {
  alternativeNotify;

  constructor(owner, args) {
    super(owner, args);
    this.alternativeNotify = Notify.create();
  }

  @action
  clicked() {
    this.alternativeNotify.success('Hello from the controller');
  }
}
```

**Comment by btecu** [comment]
Ciao @gvocale! Do you mind submitting a pull request to update the documentation?

**Comment by gvocale** [comment]
@btecu I don't seem to have write access and be able to open the PR. Can I still open a PR in some other way?

**Comment by btecu** [comment]
You have to fork the repository, then make the changes and push it. Once that's done you should be able to open a pull request.

**Comment by btecu** [comment]
Fixed in #157.
